core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/jobs/RemoteJobTracker.java

```diff
@@ -310,7 +344,11 @@ public void done(IJobChangeEvent event) {
 				}
 				params.put("state", newState.name());
 				params.put("finishDate", System.currentTimeMillis());
+				
 				update(jobId, RemoteJobEntry.WITH_DONE, params.build());
+				
+				LOG.trace("Incrementing completed job counter to {}", completedJobCounter.incrementAndGet());
```

Comment:
Please separate the counter increment to a separate line; it is easy to mistake this logging statement as a side-effect free operation:

```suggestion
				final long completedJobs = completedJobCounter.incrementAndGet();
				LOG.trace("Incrementing completed job counter to {}", completedJobs);
```